.github/workflows/env-setup-deploy-secrets.yml

```diff
@@ -47,6 +47,7 @@ jobs:
       ZEEBE_CLIENT_SECRET: ${{ secrets.ZEEBE_CLIENT_SECRET }}
       CAMUNDA_OAUTH_URL: ${{ secrets.CAMUNDA_OAUTH_URL }}
       ZEEBE_GRPC_WORKER_LONGPOLL_SECONDS: ${{ secrets.ZEEBE_GRPC_WORKER_LONGPOLL_SECONDS }}
+      QUEUE_DASHBOARD_TOKEN_SECRET: ${{ secrets.QUEUE_DASHBOARD_TOKEN_SECRET }}
```

Comment:
Following env is mising here. 

`QUEUE_DASHBOARD_BASE_URL`

`QUEUE_DASHBOARD_TOKEN_EXPIRATION_SECONDS_KEY`

Reply:
`QUEUE_DASHBOARD_TOKEN_EXPIRATION_SECONDS_KEY` is now configurable.
`QUEUE_DASHBOARD_BASE_URL` is populated by the `make` no need to set it.